examples/ssr/server.js

```diff
@@ -0,0 +1,60 @@
+import "@microsoft/fast-ssr/install-dom-shim";
+import fs from "fs";
+import { html } from "@microsoft/fast-element";
+import fastSSR from "@microsoft/fast-ssr";
+import express from "express";
+import "fast-todo-app";
+import { DefaultTodoList, app as todoApp, TodoList } from "fast-todo-app";
+
+const app = express();
+const port = 8080;
+const { templateRenderer, defaultRenderInfo } = fastSSR();
+const todoData = JSON.parse(fs.readFileSync("./todo-data.json").toString());
+
+/* eslint-disable-next-line */
+TodoList.provide(document, new DefaultTodoList(todoData));
+todoApp.define();
+
+app.use(express.static("./www"));
+
+const template = html`
+    <!DOCTYPE html>
+    <html lang="en">
+        <head>
+            <meta charset="UTF-8" />
+            <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+            <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+            <title>SSR Example</title>
+        <body>
+            <todo-app></todo-app>
+            <button id="hydrate">hydrate</button>
```

Comment:
I'm wondering if we should just remove this and instead have it auto-hydrate like most folks would want in a real app. Thoughts?

Reply:
I added this to better illustrate what the SSR package was actually doing. It's harder to see SSR "working" when we include the bundle.js automatically and immediately upgrade all the components. I'm open to removing and including the bundle automatically if the purpose of this is more to illustrate best practices though.

Reply:
Hmm. If we leave it, I think we want to comment it in some way to talk about why it's there and that you would remove it in a real solution.